src/server/game/Miscellaneous/Language.h

```diff
@@ -1203,7 +1203,8 @@ enum TrinityStrings
 
     LANG_NPCINFO_INHABIT_TYPE                     = 11008,
     LANG_NPCINFO_FLAGS_EXTRA                      = 11009,
-    LANG_INSTANCE_LOGIN_GAMEMASTER_EXCEPTION      = 11010
+    LANG_INSTANCE_LOGIN_GAMEMASTER_EXCEPTION      = 11010,
 
+    LANG_INSTANCE_BIND_MISMATCH = 11014
```

Comment:
Probably me being somewhat fussy with this, but values should be aligned with spaces. Before someone elses fusses over this :P